src/components/navigation-header/components/home-page-content/home-page-content.module.css

```diff
@@ -29,3 +29,7 @@
 		}
 	}
 }
+
+.homepageDropdownPane {
+	left: -216px;
+}
```

Comment:
Isolates this adjustment to just the homepage dropdown, instead of having it affect _all_ dropdowns across the site.